DataFormats/MuonReco/interface/Muon.h

```diff
@@ -260,7 +260,17 @@ namespace reco {
     unsigned int stationGapMaskDistance( float distanceCut = 10. ) const;
     /// same as above for given number of sigmas
     unsigned int stationGapMaskPull( float sigmaCut = 3. ) const;
-     
+    /// # of digis in a given station layer
+    /// index 0-1-2-3 = DT stations 1-2-3-4                                                             
+    /// index 4-5-6-7 = CSC stations 1-2-3-4
+    int nDigisInStation( int index) const;
```

Comment:
is there a plan to add RPC or GEMs?
perhaps a function with arguments ```MuonSubdetId, int station``` is more practical.

Reply:
I would personally not add RPC or GEM. One can disentangle the signature a shower w.r.t. some channel noise much better in a detector with many layers. My only doubt would be ME0 let me briefly discuss with MUO about this.
Anyhow OK, I will switch to a version getting `MuonSubdetId` as argument in any case.